ktxspec.adoc

```diff
@@ -1023,23 +1026,23 @@ Premultiplied Alpha::
   field if the images' RGB components have been multiplied by their
   alpha components, otherwise it must be 0.
 
-Basis Universal UASTC Format::
+Basis Universal ASTC Format::
 The Universal ASTC image format (UASTC) is indicated by `colorModel`
 `KHR_DF_MODEL_UASTC` (= 166) together with `vkFormat` `VK_FORMAT_UNDEFINED`
 (= 0). The DFD must be as described in Section 5.6.14 _KHR_DF_MODEL_UASTC_
 of <<KDF13>>. Images in this format must be transcoded to a
 GPU-supported block-compressed format or decoded to a GPU-supported
 uncompressed format before being uploaded to and sampled by a GPU.
-UASTC images can be supercompressed with Zstandard
-(`supercompressionScheme` = 2) with or without first conditioning
-the data with Rate-distortion Optimization. If supercompression is
-used, the DFD must follow the rules described in the next subsection.
+UASTC images can be supercompressed with any scheme except BasisLZ
+(`supercompressionScheme` = 1). If supercompression is used, the
```

Comment:
Seems like the new GPU Photo scheme must also be disallowed here. The spec language should be flexible enough to account for such cases.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/1f207fa65dd60a700ee88e6758d1ed67b2a8e577.